client/luigi-client.d.ts

```diff
@@ -323,6 +323,37 @@ export type addContextUpdateListener = (
 export function removeContextUpdateListener(id: string): boolean;
 export type removeContextUpdateListener = (id: string) => boolean;
 
+/**
+ * Registers a listener called when the microfrontend receives the specified custom message.
```

Comment:
```suggestion
 * Registers a listener called when the micro frontend receives a custom message.
```
I'm wondering if the word `specified` is necessary here.